src/contexts/App.tsx

```diff
@@ -28,15 +27,17 @@ export const AppProvider = ({ children }: AppProviderProps) => {
   // mapping portion of our application
   // and initialize it
   const loadMap = async () => {
-    if (state.mounted && state.container) {
-      const mapping = await import("../data/map");
-      mapping.initialize(state.container);
-    }
+    const mapping = await import("../data/map");
+    mapping.initialize(state.container);
   };
 
   useEffect(
     () => {
+      if (!state.container) {
```

Comment:
I moved this inside the effect, since it makes it more clear why it is passed in the second argument to this effect.